UPGRADE.md

```diff
@@ -8,6 +8,27 @@ awareness about deprecated code.
 
 # Upgrade to 4.4
 
+## Deprecated `AbstractSchemaManager` methods
+
+The following `AbstractSchemaManager` methods have been deprecated:
+1. `listDatabases()` - use `introspectDatabaseNames()` instead.
```

Comment:
For methods that appear in the docs, like in this example: https://github.com/doctrine/dbal/blob/2ce1218ef35cad74e488f7213df6dda701316569/docs/en/reference/schema-manager.rst?plain=1#L28-L36

should the corresponding docs be removed, or should a note be added about them being deprecated in favor of another?

Reply:
Yeah, we can replace them with the recommended ones. I recalled that I need to update the docs like 5 times but still forgot 😓